rust/src/web/mod.rs

```diff
@@ -31,6 +37,16 @@ pub async fn start_web_server<A: net::ToSocketAddrs, P: AsRef<Path>>(
             .service(blocks::get_block)
             .service(accounts::get_account)
             .service(blockchain::get_blockchain_summary)
+            .service(
+                web::resource("/graphql")
+                    .guard(guard::Post())
+                    .to(GraphQL::new(schema.clone())),
+            )
+            .service(
+                web::resource("/graphql")
+                    .guard(guard::Get())
+                    .to(index_graphiql),
+            )
```

Comment:
Should we inline the schema variable?

<img width="530" alt="image" src="https://github.com/Granola-Team/mina-indexer/assets/9849069/18bf3c8e-b0c9-455b-b441-bded843d7431">